server/rbacpolicy/rbacpolicy.go

```diff
@@ -24,6 +24,7 @@ const (
 	ResourceGPGKeys         = "gpgkeys"
 	ResourceLogs            = "logs"
 	ResourceExec            = "exec"
+	ResourceExtensions      = "extensions"
```

Comment:
RBAC docs should be updated to include this new resource. And upgrade docs should include a warning for folks who are using `*` patterns, similar to the warning when we added `exec`: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/upgrading/2.3-2.4.md#configure-rbac-to-account-for-new-exec-resource

Reply:
Added the missing rbac docs. Can you please clarify the upgrade docs requirement? I am not sure I understand the relation between the `extensions` and the `*` pattern in `exec`.

Reply:
Argo CD operators use `*` in their RBAC configuration to reduce duplication.

If we add a new capability which is matched by that `*`, an Argo CD operator might find that a minor version upgrade gives their users more capabilities than expected.

In this case, an upgrade could give users access to the extensions feature. Of course, the operator has to enable the extensions feature. But they might forget to check their RBAC rules.

To mitigate accidental increases in capabilities, we should add a note to the upgrade doc about the new RBAC resource so that Argo CD operators can proactively lock down their RBAC if they want.

Reply:
Upgrading instructions added